maxim/logger/crewai/client.py

```
@@ -708,6 +728,19 @@ def maxim_wrapper(self, *args, **kwargs):
             try:
                 # Call the original method (bound to self)
                 output = original_method.__get__(self, self.__class__)(*args, **kwargs)
+            except TypeError as e:
+                # Handle signature mismatch errors
+                if "takes from" in str(e) and "positional arguments but" in str(e):
+                    scribe().warning(f"[MaximSDK] Method signature mismatch in {final_op_name}: {e}")
+                    scribe().warning("[MaximSDK] Attempting to call original method directly...")
+                    try:
+                        # Try calling the original method directly without binding
+                        output = original_method(*args, **kwargs)
+                    except Exception as e2:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)